CHANGELOG.md

```diff
@@ -20,6 +20,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Added a template for proposing and discussing
   [enhancements](https://github.com/ComplianceAsCode/compliance-operator/tree/master/enhancements).
 
+- Added upstream documentation for `PriorityClass`
```

Comment:
We can probably omit this since we're documenting it in the enhancements section in https://github.com/ComplianceAsCode/compliance-operator/pull/71/files